substrate/frame/balances/src/lib.rs

```diff
@@ -540,6 +552,34 @@ pub mod pallet {
 				"duplicate balances in genesis."
 			);
 
+			// Generate additional dev accounts.
+			#[cfg(feature = "runtime-benchmarks")]
+			{
+				let (num_accounts, balance, ref derivation) = self.dev_accounts;
+				for index in 0..num_accounts {
+					assert!(
+						balance >= <T as Config<I>>::ExistentialDeposit::get(),
+						"the balance of any account should always be at least the existential deposit.",
+					);
+					// Check if the derivation string follows the pattern "//character/{}"
+    				assert!(
+        				derivation.starts_with("//") && 
+        				derivation[2..].contains('/') && // There should be a character after "//" and before "{}"
+        				derivation.ends_with("{}"),
+        				"Invalid derivation string"
+    				);
```

Comment:
I'd say this check is not required here.
```suggestion
```

Reply:
Oh, I see now why it is here.
It would be OK to check derivation contains '{}' (it does not need to be at the end), but I would not check derivation format.

Reply:
Yeah, Looks good.

Thanks for the review!